### DISCUSSION TITLE
Files Moved in Originals

### DISCUSSION THREAD

**Discussion by jchue** [photoprism/photoprism#1531] [Q&A] [answered]
Hello,

I've been getting started with only a few directories of photos in `originals`. Some of my directories have sub-directories containing a subset of the photos in the main directories, with the same filenames but usually `.jpg` (usually the ones I edit and export for sharing with others). For example:

```
|-originals
└---main-directory
    | img1.dng
    | img2.dng
    | img3.dng
    | img4.dng
    | img5.dng
    |
    └-sub-directory
        img2.jpg
        img3.jpg
        img4.jpg
```

When I index the files and check the Originals section in the application, I get something like this:

```
|-originals
└---main-directory
    | img1.dng
    | img5.dng
    |
    └-sub-directory
        img2.dng
        img2.jpg
        img3.dng
        img3.jpg
        img4.dng
        img4.jpg
```

First of all, I know it might not be best practice to keep the copies around. Regardless, what seems to be happening is that Photoprism is recognizing the different versions of the files and displaying them together. And I assume that it looks for any existing `.jpg` versions to be the primary, so therefore it "pulls" the `.dng` versions into the subdirectory?

Secondly, is this the expected behavior the Originals section of the application? I understand stacking elsewhere, but I would think the distinct files would be preserved as-is in Originals. Also, they're not actually displayed as stacks in Originals; they still appear separate files as I mention in the directory tree above.

Thanks!

**Comment by lastzero** [comment, marked as answer]
That's an edge case indeed as photos have a single "main" path, while stacked / related files can be physically located in different folders. The originals file browser shows folders as PhotoPrism sees them. Probably makes sense to change this to the expected behaviour like in a regular file browser that shows actual files incl. unsupported file types. Basically what you see when you open the folder via WebDAV in a compatible client like Windows Explorer or macOS Finder.

**Reply by graciousgrey** [reply]
Just added a follow up ticket for this
